pnddocs/flutter.md

```diff
@@ -20,6 +37,11 @@ In the `AppDelegate.m` add the follwing:
     return YES;
 }
 ```
+
+In order to enable Pendo pairing mode (tagging and testing) select your project select the info tab and add Url Type with pendo url scheme 
```

Comment:
To enable Pendo pairing mode (tagging and test on device) go to your application subscription, select the info tab and create a URL Type using the Pendo url scheme

Reply:
change in all places

Reply:
_go to your application subscription_ he need to go to the project In Xcode .... why application subscription?

Reply:
maybe:
In order to enable Pendo pairing mode (tagging and testing) select your project **_then_** select the info tab and add Url Type with pendo url scheme

Reply:
Sorry thought this was something else. So let's go with:

To setup the Pendo pairing mode (tagging and test on device) select your project, navigate to the relevant target, select the info tab and create a URL Type using the Pendo url scheme (found in your subscription under the App Details tab)